src/com/company/Entities/Hall.java

```diff
@@ -0,0 +1,72 @@
+package com.company.Entities;

```

Comment:
all packages should be named with lowecase and usually using singular: entity, event, service, ticket, user - also, you can group tickets and events under entity package in order to have just an entity package and a service one, and Main in root/src

Reply:
pot pastra si package-ul 'user' pe langa 'entity' si 'service' ca sa fie mai sugestiv?